Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -204,3 +269,8 @@ void PPCDebugInterface::SetPC(unsigned int address)
 void PPCDebugInterface::RunToBreakpoint()
 {
 }
+
+void PPCDebugInterface::Clear()
+{
+  ClearWatches();
```

Comment:
This seems like a weird function - shouldn't it also call `ClearAllBreakpoints` and `ClearAllMemChecks`? Is there even a use for a function that does that?

I don't see any callers, this can probably be removed.

Reply:
> the Clear() method was added to the interface to clean every debugger related features and would **at term** be used in FrameTools.cpp instead of clearing each of them one by one.

Since I'm only implementing the watches interface, I don't know if it's a good idea to put all of them in it right now. I can if that's bothering you.

Reply:
Yeah, it should implement the advertised functionality if doing so is easy before the rewrite is complete :)